docs/man/policy/registered-resources/values/update.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Update a Registered Resource Value
+command:
+  name: update
+  aliases:
+    - u
+  flags:
+    - name: id
+      shorthand: i
+      description: ID of the registered resource value to update
+    - name: value
+      shorthand: v
+      description: Optional updated value of the registered resource value (must be unique within the Registered Resource)
+    - name: action-attribute-value
+      shorthand: a
+      description: "Optional action attribute values in the format: \"<action_id>|<action_name>;<attribute_value_id|attribute_value_fqn>\""
+      default: ''
+    - name: label
+      description: "Optional metadata 'labels' in the format: key=value"
+      shorthand: l
+      default: ''
+---
+
+Update the `value` and/or metadata labels for a Registered Resource Value.
+
+If PEPs rely on this value, a value update could break access.
+
+Make sure you know what you are doing.
+
+For more information about Registered Resource Values, see the manual for the `values` subcommand.
+
+## Example
+
+```shell
```

Comment:
Could you please add some examples and document the update behavior of fully replacing or removing the action attribute values on a registered resource value?